README.md

```diff
@@ -17,7 +17,7 @@
   - under the License.
   -->
 
-# Parquet [![Build Status](https://travis-ci.org/apache/parquet-format.png?branch=master)](http://travis-ci.org/apache/parquet-format)
+# Parquet [![Build Status](https://github.com/apache/parquet-format/workflows/Test/badge.svg)](https://github.com/apache/parquet-format/actions)
```

Comment:
I believe the URL should be https://github.com/apache/parquet-format/actions/workflows/test.yml/badge.svg

(see https://github.com/apache/arrow/actions/workflows/cpp.yml/badge.svg for an example on another repo)